pkg/ebpf/extension_dummy.go

```diff
@@ -0,0 +1,69 @@
+package ebpf
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aquasecurity/tracee/common/logger"
+)
+
+// DummyExtension is a dummy extension for testing
+type DummyExtension struct {
```

Comment:
what do we need this dummy extension for?

Reply:
I'll remove it before merge ... I was using it only for testing